tls/s2n_x509_validator.h

```diff
@@ -62,6 +62,32 @@ struct s2n_x509_validator {
     int state;
 };
 
+struct s2n_x509_cert {
+    X509 *cert;
+};
+
+S2N_API
+extern int s2n_x509_cert_get_issuer_hash(struct s2n_x509_cert *cert, unsigned long *hash);
+
```

Comment:
The x509_validator module is already pretty huge and complicated. Do we want a specific crl module? s2n_crl.h?

Reply:
That makes sense! I added s2n_crl.h.